src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -161,18 +161,17 @@ This package is an internal implementation of the .NET Core SDK and is not meant
 
   <Target Name="_CreateTargetingPackArchive"
           Inputs="@(RefPackContent)"
-          Outputs="$(ArchiveOutputPath)"
+          Outputs="$(ZipArchiveOutputPath);$(TarArchiveOutputPath)"
           Condition="'$(IsPackable)' == 'true'">
     <ZipDirectory
       SourceDirectory="$(TargetingPackLayoutRoot)"
-      DestinationFile="$(ArchiveOutputPath)"
-      Overwrite="true"
-      Condition="'$(TargetRuntimeIdentifier)' == 'win-x64'" />
+      DestinationFile="$(ZipArchiveOutputPath)"
+      Overwrite="true" />
+    <!-- Requires Windows 10 version 1803 or newer -->
     <Exec
-      Command="tar -czf $(ArchiveOutputPath) ."
-      WorkingDirectory="$(TargetingPackLayoutRoot)"
-      Condition="'$(TargetRuntimeIdentifier)' == 'linux-x64'" />
-    <Message Importance="High" Text="$(MSBuildProjectName) -> $(ArchiveOutputPath)" />
+      Command="tar -czf $(TarArchiveOutputPath) ."
```

Comment:
Woah we have tar on windows now?

Reply:
Yup. curl too https://devblogs.microsoft.com/commandline/tar-and-curl-come-to-windows/